site/models/buzz-entry.php

```diff
@@ -18,6 +18,11 @@ public function isExternalLink(): bool
 			Str::startsWith($url, $this->site()->url()) === false;
 	}
 
+	public function menuUrl(): string
+	{
+		return $this->url();
+	}
+
```

Comment:
```suggestion
```

`Todo:` This looks to be the default as in the custom page method (potential default page model). So I don't think we need to add this here.